examples/pyfluent-mixing-elbow/main.py

```diff
@@ -1,176 +1,154 @@
-# # Fluent setup and solution using PyFluent APIs
-# ------------------------------------------------
+# # Integrating PyFluent with PyWorkbench: 
+# This example showcases how to use PyFluent workflow together with PyWorkbench - (Python client scripting for Ansys Workbench).
```

Comment:
```suggestion
# # Integrating PyFluent with PyWorkbench 
# This example shows how to use the PyFluent workflow with PyWorkbench - (Python client scripting for Ansys Workbench).
```
It's unclear what the content in the parentheses is supposed to convey. Can you clarify, breaking it into two sentences.